enterprise/frontend/.eslintrc

```diff
@@ -13,11 +20,15 @@
             "name": "moment-timezone",
             "message": "Moment is deprecated, please use dayjs"
           },
+          {
+            "name": "@storybook/testing-library",
```

Comment:
we don't seem to have this anymore. Can you double-check? I think it's changed to `@storybook/test` after we upgraded to Storybook 8.